.github/workflows/update-swc.yml

```diff
@@ -0,0 +1,65 @@
+name: Update SWC and Build WASM
+
+on:
+  workflow_dispatch:
+    inputs:
+      swc_version:
+        description: 'SWC version to update to (e.g., 1.7.5)'
+        required: false
+  schedule:
+    - cron: '0 0 * * 1' # Every Monday at 00:00 UTC
+
+env:
+    NODE_VERSION: lts/*
+
+jobs:
+  update-swc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+
+      - name: Get latest SWC version
+        id: check-swc-version
+        run: |
+            SWC_VERSION=$(npm view @swc/core version)
+            echo "SWC_VERSION=$SWC_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Get current SWC version
+        run: |
+            SWC_VERSION=$(cat lib/swc/package.json | jq -r '.version')
+            echo "SWC_VERSION=$SWC_VERSION" >> $GITHUB_ENV
+            if [[ $SWC_VERSION == ${{ steps.check-swc-version.outputs.SWC_VERSION }} ]]; then
+              echo "SWC is already up-to-date. Exiting."
+              exit 0
+            fi
+      - name: Git config
+        run: |
```

Comment:
we should use node.js github bot here, should be the same on nodejs/node https://github.com/nodejs/node/blob/76c8ba9d9b71eee2b44f8097820b885e2a85ddd2/.github/workflows/tools.yml#L351

Reply:
Should have check main repo first my bad, fixed